src/util/color.h

```diff
@@ -211,15 +209,17 @@ namespace Color {
         return QObject::tr("Undefined Color");
     };
 
-    // Returns a new default colors representation, i.e. maps each default color to itself.
+    // The default colors representation, i.e. maps each default color to itself.
     // Stores the color's name() property, e.g. "#A9A9A9"
-    static std::unique_ptr<ColorsRepresentation> makeDefaultRepresentation() {
-        std::unique_ptr<ColorsRepresentation> representation = std::make_unique<ColorsRepresentation>();
-        for (QColor color : predefinedColors()) {
-            representation->setRepresentation(color, color);
+    //
+    // It's fast to copy the default representation. See comment on ColorsRepresentation.
+    static const ColorsRepresentation defaultRepresentation = [](){
```

Comment:
What exactly is this? I have never seen this syntax (`[](){}();`)

Reply:
It's a [lambda function](https://en.cppreference.com/w/cpp/language/lambda). In this context means: "execute this function once and initialize the var with its result."
If `Color` was a class instead of a namespace we could just initialize the var in its constructor.

Reply:
Thank you, I knew what lambdas are but not how they look in C++.